src/shapes/Image.ts

```diff
@@ -32,10 +49,30 @@ export const imageDefaultValues: Partial<TClassProperties<Image>> = {
   imageSmoothing: true,
 };
 
+export interface SerializedImageProps
+  extends SerializedObjectProps {
+    src: string;
+    crossOrigin: string | null;
+    filters: any[];
+    resizeFilter?: any;
+    cropX: number;
+    cropY: number;
+  };
```

Comment:
probably those will be similar in content, but different in types.
i think both crossOrigin and filters apply to both